package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "uids",
   "type": "module",
-  "version": "4.0.0-alpha1",
+  "version": "4.0.0-alpha2",
```

To actually generate the assets that will be linked in the docs, we have to create a new release. If we are going to create a new release, we need to update the version, since we already have a release for [`4.0.0-alpha2`](https://github.com/uiowa/uids/releases/tag/v4.0.0-alpha2)
```suggestion
  "version": "4.0.0-alpha3",
```